plugin.json

```diff
@@ -5,6 +5,7 @@
     "dependencies": ["worker"],
     "npm": {
         "dependencies": {
+            "reconcile.js": "github:matthewma7/reconcile.js"
```

Comment:
This is a library that comes from your personal github?  I don't feel good about relying on that.  What does the library do?  Any other ideas about how to bring it in?

Reply:
This is a library that I start using in Minerva recently. It helps update the DOM without destroying the old DOM and without need manual jQuery manipulation. This library is particularly useful here is that If I re-render with destructive backbone render(), the <video> will lose all its states(play/stop, seek position etc). 
It is now coming from my fork of the library, but it doesn't need to anymore because the feature I need is published to NPM.
Overall, this library gives me a React like Event -> State change -> Update DOM cycle, which I found really helpful when working with backbone. 
Hope what I just said is make sense. I can talk about this in detail some time.